paasta_tools/mesos_maintenance.py

```diff
@@ -200,6 +200,49 @@ def is_host_down(hostname=getfqdn()):
     return hostname in get_down_hosts()
 
 
+def get_hosts_forgotten_draining():
+    """Find hosts that are still marked as draining (rather than down) after the start
+    of their maintenance window. For each of these hosts, either mark them as 'down' for
+    maintenance, adjust the time of their maintenance window, or cancel the maintenance
+    window if it is no longer needed.
+    :returns: a list of hostnames of hosts forgotten draining
```

Comment:
The docs are accurate, but they would be better suited as a `Tip` on a `monitoring_check` than in the function definition. They describe actions a human operator can take to deal with hosts that are forgotten draining. I'll move them.